### PR TITLE
Adding cronjobs/jobs/daemonsets to the output

### PR DESCRIPTION
**Pull request by thekad**
Might be useful for cases when e.g. etcd-backup is stuck

```release-note
NONE
```

### CODE REVIEW

**Comment by thekad** [comment]
flake #1778 

/retest

File: pkg/vmimage/data/usr/local/bin/oc-get-all [hunk 1]
```diff
@@ -7,7 +7,7 @@ then
   exit
 fi
 
-for target in group user deploy service pod route
+for target in node group user daemonset service route deploy cronjob job pod
```
Comment (ehashman) [inline pkg/vmimage/data/usr/local/bin/oc-get-all]: Are cronjobs different than jobs?

Comment (thekad) [reply pkg/vmimage/data/usr/local/bin/oc-get-all]: yes, a job named "foo" will run once and you have to re-create again if you want it to run again. A cronjob named "bar" will spawn jobs named "bar-${timestamp}" on whatever schedule the parent cronjob has defined.

**Pull request merged**